2022 edition (#4)
go-devroom/cfp: README.md

Hi gophers,

The Go community has been represented at FOSDEM every year since 2014. The year 2022 won't be any different

This year again the Go devroom will be **online** on Saturday February 5th 2022 and we are now looking for speakers. The format will be 25 minutes talk with 5 minutes for Q&A at the end.
All talks will be presented online either pre-recorded or live (which one you prefer to give is up to you! We do however require a backup video to be uploaded).
During the talks we will have an online Q&A and chatroom moderated by our friendly volunteers helping to provide a fun online experience.

This is a great opportunity for new speakers, and we will have people helping you craft your talk and slides if so you wish. So if you, or someone you know, are working on something cool that might be good on a virtual stage, submit a proposal!

> Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/event/new). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2022.

Author: ```suggestion
Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/event/new). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 20th 2022.
```

need editing time

> Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/submission/FOSDEM22). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2022.

Author: ```suggestion
Do you want to be one of our speakers? Please submit your proposal using [pentabarf](https://penta.fosdem.org/event/new). The deadline for submission is Decemnber 30th 2021 (23:59 UTC). If you are selected we would like to have the (backup) video uploaded by January 31st 2022.
```

Cheers,

> Maartje Eyskens

Reviewer: ```suggestion
Maartje Eyskens
Kris Nóva
```
